Functions/Quantification/xASL_quant_DefineParameters.m

```diff
@@ -0,0 +1,164 @@
+function [x] = xASL_quant_DefineParameters(x)
```

Comment:
Is that really the best name?
Shall we call them physiological parameters or quantification parameters

Reply:
I agree, but I thought that is implied in "quant". I will change it to xASL_quant_DefineQuantificationParameters. (BTW they are not physiological parameters but physics parameters right)